### ISSUE TITLE
About Overfitting 

### ISSUE THREAD

**Issue by lsj2004** [X-niper/UniTalker#32]
Thanks for your awesome job! However, I've encountered a problem: I want to add a flame parameter (only expression + jaw) MSE loss to the original facial vertices loss when training the D5 dataset. However, no matter how I adjust the parameters, the training loss keeps decreasing slowly, while the validation and test losses keep fluctuating with no downward trend. It seems like overfitting is happening. Do you have any suggestions for me?
